src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -100,7 +103,7 @@ public async Task<OrganizationUserDetailsResponseModel> Get(string id, bool incl
         var response = new OrganizationUserDetailsResponseModel(organizationUser.Item1, organizationUser.Item2);
 
         // Downgrade Custom users with no other permissions than 'Edit/Delete Assigned Collections' to User
-        response.Type = GetFlexibleCollectionsUserType(response.Type, response.Permissions);
+        response.Type.GetFlexibleCollectionsUserType(response.Permissions);
```

Comment:
You're no longer doing anything with this return value.

Reply:
haha yea I missed that part. Thanks.